google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/UpdateTableRequest.java

```diff
@@ -88,11 +95,11 @@ public boolean equals(Object o) {
     if (this == o) return true;
     if (!(o instanceof UpdateTableRequest)) return false;
     UpdateTableRequest that = (UpdateTableRequest) o;
-    return Objects.equals(requestBuilder, that.requestBuilder);
+    return Objects.equals(requestBuilder.build(), that.requestBuilder.build());
```

Comment:
why changing this?

Reply:
I can remove it but that's what we do in other like CreateTableRequest